.gdbinit

```diff
@@ -51,13 +58,100 @@ end
 
 document print_cvs
 	Prints the compiled variables and their values.
-	If a zend_execute_data pointer is set this will print the compiled
-	variables of that scope. If no parameter is used it will use
+	If a zend_execute_data pointer is set, this will print the compiled
+	variables of that scope. If no parameter is used, it will use
 	current_execute_data for scope.
 
-	usage: print_cvs [zend_execute_data *]
+	Usage: print_cvs [(zend_execute_data*)ped]
+end
+
+# $arg0: zend_execute_data*
+#        if not specified, try to use current execute_data
+define print_tmps
```

Comment:
Hmm, many of those will not be live (as in contain trash) so I wonder how useful this actually is, and may probably also crash due to memory corruption.